python/pfs/drp/stella/fitPfsFluxReference.py

```diff
@@ -149,7 +149,7 @@ def run(self, pfsConfig, pfsMerged):
         pfsConfig : `pfs.datamodel.pfsConfig.PfsConfig`
             Configuration of the PFS top-end,
             in which information of broad band fluxes count.
-        pfsMerged : `pfs.datamodel.pfsFiberArraySet.PfsFiberArraySet`
+        pfsMerged : `pfs.drp.stella.PfsFiberArraySet`
```

Comment:
Isn't it `pfs.drp.stella.datamodel.PfsFiberArraySet`?
I think this change should be a separate commit, as it's not relevant to the normalisation change.

Similarly throughout.

Reply:
I am splitting the commit into two. In the first commit `pfs.datamodel.PfsFiberArraySet` will be replaced by `pfs.drp.stella.datamodel.PfsFiberArraySet`, and in the second commit the normalisation will be done.